resources/helm/v2.1/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -81,7 +111,7 @@ template: |
     {{- end }}
   {{- else }}
     {{- if .Values.global.proxy.resources }}
-      {{ toYaml .Values.global.proxy.resources | indent 4 }}
```

Comment:
interesting. this change suggests that proxy resource config didn't work previously?